README.md

```diff
@@ -104,28 +103,7 @@ wget -O ~/.paloma/config/addrbook.json https://raw.githubusercontent.com/palomac
 service palomad start
 ```
 
-### Upgrading from `paloma-testnet-7` to `paloma-testnet-8`
-1. Stop your paloma version and get 0.7.0
-```
-service palomad stop
-wget -O - https://github.com/palomachain/paloma/releases/download/v0.7.0/paloma_Linux_x86_64.tar.gz | \
-  tar -C /usr/local/bin -xvzf - palomad
-```
-
-2. Reset your local chain state:
-```bash
-palomad tendermint unsafe-reset-all --home $HOME/.paloma
-```
-
-3. Copy the latest genesis and addrbook
-```shell
-wget -O ~/.paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/testnet/master/paloma-testnet-8/genesis.json
-wget -O ~/.paloma/config/addrbook.json https://raw.githubusercontent.com/palomachain/testnet/master/paloma-testnet-8/addrbook.json
-```
-
-4. Ensure that the [latest pigeon](https://github.com/palomachain/pigeon#install) is up & running and that you have at least 0.1 ETH on eth mainnet target chain. Make sure to change the `chain-id: paloma-testnet-7` to `chain-id: paloma-testnet-8` in your pigeon config yaml file ([example](https://github.com/palomachain/pigeon/pull/51/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R85)).
-
-5. Start the palomad
+7. If you are planning to be a **VALIDATOR with stake**, ensure that your pigeon relayer is up & running and that you have at least 0.1 ETH on eth mainnet target chain address that is in your pigeon configuration file.
```

Comment:
Why are all those lines deleted?

Reply:
Because those lines are already covered in the pigeon README instructions here: https://github.com/palomachain/pigeon#config-setup